recipes/pydownsampler/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pydownsampler" %}
+{% set version = "1.0" %}
+{% set sha256 = "0c7d5515382f343a058155d713376d9f6d7245d8dcfa603d9dd3acc70c2643d2" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pydownsampler = pydownsampler.pydownsampler:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - docopt
```

Comment:
I think you can remove everything here, but pip and python

Reply:
Fixed